mne/io/tests/test_utils.py

```diff
@@ -9,11 +9,11 @@
 
 def test_check_orig_units():
     """Test the checking of original units."""
-    orig_units = dict(FC1='nV', Hfp3erz='n/a', Pz='uV', greekMu=u'μV',
-                      microSign=u'µV')
+    orig_units = dict(FC1='nV', Hfp3erz='n/a', Pz='uV', greekMu='μV',
+                      microSign='µV')
     orig_units = _check_orig_units(orig_units)
     assert orig_units['FC1'] == 'nV'
     assert orig_units['Hfp3erz'] == 'n/a'
-    assert orig_units['Pz'] == u'µV'
-    assert orig_units['greekMu'] == u'µV'
-    assert orig_units['microSign'] == u'µV'
+    assert orig_units['Pz'] == 'µV'
+    assert orig_units['greekMu'] == 'µV'
+    assert orig_units['microSign'] == 'µV'
```

Comment:
here also I'm only removing unicode string designator, not changing string content.